api/execute.go

```diff
@@ -52,7 +52,7 @@ func (e *taskExecutor) checkServiceStatus(s *service.Service) error {
 		return err
 	}
 	if status != service.RUNNING {
-		return &NotRunningServiceError{ServiceID: s.ID}
+		return &NotRunningServiceError{ServiceID: s.Hash}
```

Comment:
here we use _ServiceID_ it should be _ServiceHash_ or _Hash_.

Reply:
Let's leave it like this. From a user perspective, ServiceID is fine.
I'm also preparing another PR that remove completely Hash.